types/coordenadas-do-cep/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ 
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove the `rules` property and, if there are lint errors, add line-level suppressions.